### DISCUSSION TITLE
Prevent automatic third party requests in videos

### DISCUSSION THREAD

**Discussion by Lednerb** [Lednerb/bilberry-hugo-theme#318] [Ideas]
For the `video` post type, the following feature should be implemented:

A new privacy-protecting function should be implemented to show only the video poster and only if a user clicks on the poster the associated `iframe` should be loaded.

Furthermore the video-url should be sufficient, the logic behind (if vimeo or youtube should be loaded) can be implemented in the template directly, so there will be the parameters:

- `videoPoster`
- `videoUrl`

This can be implemented to be compatible with the actual parameter names, preventing a breaking change here.

**Comment by RoneoOrg** [comment]
Related discussion on gohugo.io: https://discourse.gohugo.io/t/embedding-youtube-in-a-privacy-friendly-way/35784/8
